bench/actix-rs/optimized/2wg1z1eatrkafji9.ll

```diff
@@ -2077,9 +2077,10 @@ define internal fastcc void @"_ZN4core3ptr365drop_in_place$LT$hashbrown..scopegu
 
 2:                                                ; preds = %13, %0
   %.sroa.0.08.i.i = phi i64 [ 0, %0 ], [ %spec.select6.i.i, %13 ]
-  %3 = icmp ult i64 %.sroa.0.08.i.i, %.0.val
-  %4 = zext i1 %3 to i64
-  %spec.select6.i.i = add nuw i64 %.sroa.0.08.i.i, %4
+  %3 = icmp uge i64 %.sroa.0.08.i.i, %.0.val
+  %4 = xor i1 %3, true
+  %spec.select6.i.i = zext i1 %not..i.i to i64
+  %spec.select6.i.i = add nuw i64 %.sroa.0.07.i.i, %4
```

Comment:
this can not be good that the same name is used twice

Reply:
Please refer to the updated issue description in https://github.com/dtcxzyw/llvm-opt-benchmark/issues/1312:
> Update (2025-02-01): Unrelated diffs have been significantly reduced with a heuristic-based value name remapping algorithm (Example: https://github.com/dtcxzyw/llvm-opt-benchmark/pull/2037 -> https://github.com/dtcxzyw/llvm-opt-benchmark/pull/2055). Note that the post-processed IR may be incorrect or invalid. Please refer to the previous commit if you are confused by the result.